packages/forms/src/radioCard/FormikRadioCardField.tsx

```diff
@@ -0,0 +1,68 @@
+import {
+  RadioCardItemProps,
+  RadioFieldCard,
+  RadioGroupField,
+  RadioGroupFieldProps,
+  Stack,
+} from '@superdispatch/ui';
+import { FieldValidator, useField, useFormikContext } from 'formik';
+import { forwardRef, ForwardRefExoticComponent } from 'react';
+
+export interface FormikRadioCardFieldProps
+  extends Omit<RadioGroupFieldProps, 'error' | 'value'> {
+  name: string;
+  validate?: FieldValidator;
+  radioItems: RadioCardItemProps[];
+}
+
+export const FormikRadioCardField: ForwardRefExoticComponent<FormikRadioCardFieldProps> =
+  forwardRef(
+    (
+      {
+        name,
+        validate,
+        radioItems,
+
+        onBlur,
+        onChange,
+        disabled,
+        helperText,
```

Comment:
you are extracting the helper text but not using it?

Reply:
removed helperText